app/onerror.js

```diff
@@ -54,10 +56,9 @@ module.exports = function myErrorHandler(errorMessage, fileUrl, lineNumber, coln
 
     html = [
     '<div style="background: #fefefe;border: gray solid 1px;margin-left: -200px;position: fixed;left: 50%;top: 20%;z-index: 11;width: 390px;padding: 20px 25px;padding-top:30px">',
-      '<p>' + chosenMessage + '</p>',
-      '<p>We were unable to log this error to our server so could you please send us a note at <a style="text-decoration: underline;" href="mailto:support@tidepool.org">support@tidepool.org</a> and we\'ll try to see what broke?</p>',
```

Comment:
So the thing is - it'd actually be helpful for us to prompt for a screenshot almost all the time, y/y? Because having the UTC timestamp will help us pin to the logs so much easier.